test/lib/ufe/testDuplicateCmd.py

```diff
@@ -646,7 +646,85 @@ def testPrimWithPayload(self):
         rootLayerText = rootLayer.ExportToString()
         self.assertNotIn('"Geom"', rootLayerText)
         self.assertNotIn('Mesh', rootLayerText)
-    
+
+    def testDuplicateUniqueName(self):
+        '''Test the duplicate of prims and ensure the new name follows Maya
+        unique new name standard.'''
```

Comment:
Add a bunch of new test cases to verify my code changes for Usd prims, follow how Maya naming works.